src/makeRobot.js

```diff
@@ -38,7 +38,58 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    'name': name,
+    'version': version,
+    'wheels': wheels,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
+      return `name: ${this.name
+      }, chip version: ${this.version
+      }, wheels: ${this.wheels}`;
```

Comment:
Тут лучше использовать конкатенацию строк:
```suggestion
      return `name: ${this.name}, `
      + `chip version: ${this.version}, `
      + `wheels: ${this.wheels}`;
```